.github/workflows/run_ock_external_tests.yml

```diff
@@ -174,7 +174,7 @@ jobs:
           llvm_source: ${{ inputs.llvm_source }}
 
   build_icd:
-    if: inputs.ock && ( inputs.test_tornado || inputs.test_opencl_cts || inputs.test_sycl_cts || inputs.test_sycl_e2e)
+    if: inputs.ock && ( inputs.test_tornado || inputs.test_opencl_cts || inputs.test_sycl_cts || inputs.test_sycl_e2e) && contains(inputs.target_list, 'linux')
```

Comment:
It's not clear to me whether this is part of what will be reverted before merging. If it's not: I would expect this check to always return `false`.